namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ObserverCache.scala

```diff
@@ -1,10 +1,12 @@
 package io.buoyant.namerd.iface
 
-import com.google.common.cache.{CacheBuilder, RemovalListener, RemovalNotification}
```

Comment:
please format imports in alphabetical order with no blank lines

Reply:
Ok